clean/new_orleans_pd_uof.py

```diff
@@ -0,0 +1,552 @@
+        pd.read_csv(deba.data("raw/new_orleans_pd/new_orleans_pd_uof_2016_2021.csv"))
+        .pipe(clean_column_names)
+        .rename(
+            columns={
+                "occurred_date": "occur_date",
+                "shift": "shift_time",
+                "subject_ethnicity": "citizen_race",
+                "subject_gender": "citizen_sex",
+                "subject_hospitalized": "citizen_hospitalized",
+                "subject_injured": "citizen_injured",
+                "subject_influencing_factors": "citizen_influencing_factors",
+                "subject_distance_from_officer": "citizen_distance_from_officer",
+                "subject_age": "citizen_age",
+                "subject_build": "citizen_build",
+                "subject_arrested": "citizen_arrested",
+                "subject_arrest_charges": "citizen_arrest_charges",
+            }
+        )
+        .drop(
+            columns=[
+                "officer_years_of_service",
+                "officer_name",
+                "officer_race_ethnicity",
+                "officer_gender",
+                "officer_age",
+                "officer_injured",
+            ]
+        )
+        .pipe(join_uof_columns)
+        .pipe(clean_originating_bureau)
+        .pipe(clean_division)
+        .pipe(clean_division_level)
+        .pipe(clean_unit)
+        .pipe(clean_weather_condition)
+        .pipe(clean_service_type)
+        .pipe(clean_disposition)
+        .pipe(clean_working_status)
+        .pipe(clean_light_condition)
+        .pipe(clean_use_of_force_reason)
+        .pipe(clean_use_of_force_type)
+        .pipe(clean_tracking_number)
+        .pipe(
+            standardize_desc_cols,
+            [
+                "shift_time",
+                "investigation_status",
+                "use_of_force_reason",
+                "use_of_force_type",
+                "use_of_force_level",
+                "use_of_force_effective",
+            ],
+        )
+        .pipe(set_values, {"agency": "New Orleans PD"})
+        .pipe(
+            gen_uid,
+            [
+                "occur_date",
+                "originating_bureau",
+                "division_level",
+                "division",
+                "unit",
+                "working_status",
+                "shift_time",
+                "investigation_status",
+                "disposition",
+                "service_type",
+                "light_condition",
+                "weather_condition",
+                "use_of_force_reason",
+                "use_of_force_type",
+                "use_of_force_level",
+                "use_of_force_effective",
+                "tracking_number",
+                "agency",
+            ],
+            "uof_uid",
+        )
+        .dropna(subset=["uof_uid"])
+        .drop_duplicates(subset=["uof_uid"])
+    )
+    return df
+
+
+if __name__ == "__main__":
+    uof_citizen = clean_citizen()
+    uof_officer = clean_officer()
+    uof = clean_uof()
```

Comment:
This is not super efficient. Ideally every column should only be dropped, or renamed, or cleaned only once. So something like this is more ideal:
```python
uof = clean_uof()
# extract citizen columns into a separate dataframe and remove citizen columns from uof
uof_citizen, uof = extract_citizen(uof)
uof_officer, uof = extract_officer(uof)

Reply:
Got it. Thanks.